packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -83,6 +83,8 @@ library LibVerifying {
         // Retrieve the latest verified block and the associated transition used
         // for its verification.
         TaikoData.SlotB memory b = state.slotB;
+        if (b.provingPaused) return;
```

Comment:
@Brechtpd if proving is paused, verification is effectively paused as well, but we didn't revert here, simply return (doing nothing)